### ISSUE TITLE
FS Chat: Blur not working (Firefox)

### ISSUE THREAD

**Issue by Munzu** [FrankerFaceZ/Add-Ons#50]
The background color blur doesn't work for me, no matter how high I set the blur strength. 
I'm using the newest Firefox version.

Please let me know if you need further info.

Thanks in advance

**Comment by SirStendec** [comment]
Unfortunately, Firefox does not support CSS `backdrop-filter` which is used for the blurring effect. I thought I had remembered to gate the setting behind a `CSS.supports()` call, but apparently I did not.

There are flags to enable Firefox support, but I generally do not recommend users to mess with advanced feature settings unless they know what they're doing. Features not enabled by default are often not ready for widespread use.

Details: https://developer.mozilla.org/en-US/docs/Web/CSS/backdrop-filter#browser_compatibility

I'll leave this issue open until I fix the setting being exposed when not supported.
